tests/nightly/test_large_array.py

```diff
@@ -279,6 +279,19 @@ def test_diag():
     assert_almost_equal(r.asnumpy(), np.diag(a_np, k=k))
 
 
+def test_transpose():
+    a = nd.arange(0, LARGE_X).reshape(LARGE_X, 1)
+    b = nd.broadcast_to(a, shape=(a.shape[0], SMALL_Y))
+    t = b.T
+    assert t.shape == (SMALL_Y, LARGE_X)
+    assert np.sum(t[:,-1].asnumpy() == LARGE_X) == b.shape[1]
+    t = nd.swapaxes(b, dim1=0, dim2=1)
```

Comment:
test `transpose` instead?

Reply:
These are 3 different transpose operations all three are tested

Reply:
I thought `transpose` is a operator name? If you are testing `swapaxes`, maybe change the test name as well?

Reply:
flip(), swapaxes() and b.T are all transpose operations